### PR TITLE
Karina/wmdp 116 contact info

### PR DESCRIPTION
**Pull request by karinamzalez**
## Ticket

https://wicmtdp.atlassian.net/browse/116

- contact page
- textinput component

- phone number input should format numbers and not allow non-numbers 

Screen Shot 2022-07-18 at 12 11 46 PM

### CODE REVIEW

File: app/src/pages/contact.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        handleChange={handleChange}
+        id="lastName"
+        label={t('Contact.lastName')}
+        value={form.lastName}
+      />
+      <br />
+      <Label className="usa-label" htmlFor="phone">
+        {t('Contact.phone')}
+      </Label>
+      <PhoneInput
```
Comment (rocketnova) [inline app/src/pages/contact.tsx:49]: Out of curiosity, where does this get imported into the file? 👀 How come this doesn't throw an error?

File: app/src/pages/contact.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        format="###-###-####"
+        mask="_"
+        role="textbox"
+        className="usa-input"
+        id="phone"
+        value={form.phone}
+        onChange={handleChange}
+      />
+      <Helper>{t('Contact.phoneHelper')}</Helper>
```
Comment (rocketnova) [inline app/src/pages/contact.tsx:58]: Same with `Helper` 👀

File: app/public/locales/en/common.json [hunk 3]
```diff
@@ -1,4 +1,12 @@
 {
+  "Contact": {
```
Comment (rocketnova) [inline app/public/locales/en/common.json:2]: I think we're missing the `/es/common.json` translations for these.

**Review by rocketnova — APPROVED** [review]
Looks great! 🚢

File: app/src/pages/contact.tsx (hunk 1 above)
Comment (karinamzalez) [reply app/src/pages/contact.tsx:49]: PhoneInput and Helper are styled-components defined at the bottom of this file :)

**Pull request merged**